plugins/modules/ilo_redfish_command.py

```diff
@@ -0,0 +1,385 @@
+        type: int
+  service_attributes:
+    required: false
+    description:
+      - service BIOS attributes that needs to be verified in the given server
+    type: dict
+  uefi_boot_order:
+    required: false
+    description:
+      - Input UEFI Boot Order
+    type: list
+    elements: str
+  logical_drives_names:
+    required: false
+    description:
+      - List of names of logical drives to be fetched from the given server
+    type: list
+    elements: str
+  cert_file:
+    required: false
+    description:
+      - absolute path to the server cert file
+    type: str
+  key_file:
+    required: false
+    description:
+      - absolute path to the server key file
+    type: str
+author:
+  - Gayathiri Devi Ramasamy (@Gayathirideviramasamy)
+  - T S Kushal (@TSKushal)
+  - Varni H P (@varini-hp)
+  - Prativa Nayak (@prativa-n)
+'''
+
+EXAMPLES = '''
+  - name: Verify service BIOS attributes
+    community.general.ilo_redfish_command:
+      category: Systems
+      command: VerifyServiceBiosAttributes
+      baseuri: "***.***.***.***"
+      username: "abcxyz"
+      password: "******"
+      service_attributes:
+        ConnectSequenceTrace: "Disabled"
+        CustomDebugBlkIo: "Disabled"
+
+  - name: Verify bios attributes
+    community.general.ilo_redfish_command:
+      category: Systems
+      command: VerifyBiosAttributes
+      baseuri: "***.***.***.***"
+      username: "abcxyz"
+      password: "*****"
+      bios_attributes:
+        SubNumaClustering: "Disabled"
+        WorkloadProfile: "Virtualization-MaxPerformance"
+
+  - name: Verify logical drives
+    community.general.ilo_redfish_command:
+      category: Systems
+      command: VerifyLogicalDrives
+      baseuri: "***.***.***.***"
+      username: "abcxyz"
+      password: "******"
+      raid_details:
+        - LogicalDriveName: LD1
+          Raid: Raid1
+          DataDrives:
+              DataDriveCount: 2
+              DataDriveMediaType: HDD
+              DataDriveInterfaceType: SAS
+              DataDriveMinimumSizeGiB: 0
+
+  - name: Verify UEFI boot order
+    community.general.ilo_redfish_command:
+      category: Systems
+      command: VerifyUefiBootOrder
+      baseuri: "***.***.***.***"
+      username: "abcxyz"
+      password: "******"
+      uefi_boot_order: ["Generic.USB.1.1"]
+
+  - name: Verify specified logical drives
+    community.general.ilo_redfish_command:
+      category: Systems
+      command: VerifySpecifiedLogicalDrives
+      baseuri: "***.***.***.***"
+      username: "abcxyz"
+      password: "******"
+      raid_details:
+        - LogicalDriveName: LD1
+          Raid: Raid1
+          DataDrives:
+              DataDriveCount: 2
+              DataDriveMediaType: HDD
+              DataDriveInterfaceType: SAS
+              DataDriveMinimumSizeGiB: 0
+
+  - name: Check server reboot status
+    community.general.ilo_redfish_command:
+      category: Systems
+      command: CheckRebootStatus
```

Comment:
Can we pick a different name for this command? The naming sounds like it should be standard Redfish, but looking at what the command does, it goes off to OEM properties. I'd like to prevent potential confusion with users thinking this might be something standard.

Reply:
Does "CheckiLORebootStatus" sound good?

Reply:
Resolving this conversation, Thanks!